_includes/sub-template-prerequisites-storage.md

```diff
@@ -0,0 +1,11 @@
+### Azure Storage account 
+
+If you are deploying a Java EE application or using your own JDBC datasource driver, you are required to 
+have application packages and jdbc libraries uploaded to a blob storage container in an Azure Storage Account.
+
+To create Azure Storage Account and blobs, follow the steps in [Quickstart: Upload, download, and list blobs with the Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal).
+
+Upload your application packages (.jar, .war, .ear files) to the blob.
+
+Upload your jdbc drivers (.jar files) to the blob.
```

Comment:
jdbc -> JDBC

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb